use glog through cmake config files

this sets the required defines and makes glog work properly

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 59e2ec1..8575c78 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,7 +56,7 @@ if (TRUST_STORE_MIR_AGENT_ENABLED)
   )
 endif()
 
-pkg_check_modules(GLOG libglog REQUIRED)
+find_package(glog 0.7.1 REQUIRED)
 pkg_check_modules(PROCESS_CPP process-cpp REQUIRED)
 
 include(CTest)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index ac87e7f..1d05d2e 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -207,11 +207,11 @@ target_link_libraries(
 
   dbus-cpp
   xdg
+  glog::glog
 
   ${Boost_LIBRARIES}
   ${DBUS_LIBRARIES}
   ${GFLAGS_LDFLAGS}
-  ${GLOG_LDFLAGS}
   ${GLIB_LDFLAGS}
   ${GOBJECT_LDFLAGS}
   ${LIBAPPARMOR_LDFLAGS}
